src/app/layout.tsx

```diff
@@ -16,7 +18,15 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="en">
-      <body className={inter.className}>{children}</body>
+      <body className={inter.className}>
+        <div className="flex flex-col">
```

Comment:
As ur children element is block element which means it will be display in col only so flex & flex-col is unnecessary here

Reply:
Header needs to be given position `sticky` so needed flex display